docs/best-practices/routing/routing.md

```diff
@@ -2,13 +2,13 @@
 
 ## Routing Mode
 
-First, you should determine if you are going with [HTML 5 history mode or hash mode](https://router.vuejs.org/guide/essentials/history-mode.html) It is 2018, and I definitely recommend that you use HTML5 history mode.
+First, you should determine if you are going with [HTML 5 history mode or hash mode](https://router.vuejs.org/guide/essentials/history-mode.html) It is 2020, and we definitely recommend that you use HTML5 history mode.
```

Comment:
Could we change this to be more dynamic? I think html DOM approach will work, but let's search for other option before using it.

Reply:
will just get rid off Routing Mode, are you fine with this?

Reply:
I’m referring to the year, you updated the year for the current one but as long as it changes we’ll have to change the code. Ideally to have it dynamically built

Reply:
The section was updated